### PR TITLE
Added Diameter S13 Resource Adaptor

### PR DESCRIPTION
**Pull request by richardgood**
I have added the Diameter S13 Resource Adaptor:

The S13 interface shall enable the ME Identity check procedure between the MME and the EIR as described in the 3GPP TS 23.401

### CODE REVIEW

**Comment by deruelle** [comment]
Thanks @richardgood ! @brainslog can you review ?

File: resources/diameter-s13/du/pom.xml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.mobicents.servers.diameter.resources</groupId>
+		<artifactId>s13-parent</artifactId>
```
Comment (SergeyLee) [inline resources/diameter-s13/du/pom.xml]: Please use <artifactId>restcomm-slee-ra-diameter-s13</artifactId> and fix artifacts for another modules
Please see s6a RA for example.

File: resources/diameter-s13/du/pom.xml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+		<version>1.0.0.FINAL</version>
```
Comment (SergeyLee) [inline resources/diameter-s13/du/pom.xml]: Please use <version>2.8.0-SNAPSHOT</version>
and add <relativePath>../..</relativePath>

File: resources/diameter-s13/du/pom.xml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+	</parent>
+
+	<artifactId>s13-ra-DU</artifactId>
```
Comment (SergeyLee) [inline resources/diameter-s13/du/pom.xml]: Use these tags:

```
<groupId>org.mobicents.resources</groupId>
<artifactId>restcomm-slee-ra-diameter-s13</artifactId>
```

File: resources/diameter-s13/du/pom.xml [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+	<name>Mobicents :: Diameter Resources :: ${pom.artifactId}</name>
```
Comment (SergeyLee) [inline resources/diameter-s13/du/pom.xml]: Also you should rebrand Mobicents to Restcomm in all files.

File: resources/diameter-s13/pom.xml [hunk 5]
```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <packaging>pom</packaging>
+
+    <parent>
```
Comment (SergeyLee) [inline resources/diameter-s13/pom.xml]: Please use parent like this:

```
    <parent>
        <groupId>org.mobicents.servers.jainslee.diameter</groupId>
        <artifactId>restcomm-slee-diameter</artifactId>
        <version>2.8.0-SNAPSHOT</version>
        <relativePath>../..</relativePath>
    </parent>
```

File: resources/diameter-s13/pom.xml [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+            <name>Paul Carter-Brown</name>
+            <email>paul.carter-brown@smilecoms.com</email>
+            <organization>Smile Communications</organization>
+            <organizationUrl>http://www.smilecoms.com</organizationUrl>
+        </developer>
+    </developers>
+
+    <properties>
+        <!-- Mobicents Diameter Components Versions -->
+        <mobicents.diameter.resources.base-common.version>2.7.0.FINAL</mobicents.diameter.resources.base-common.version>
```
Comment (SergeyLee) [inline resources/diameter-s13/pom.xml]: Please remove this, we use project.version for org.mobicents.resources dependencies.

**Comment by SergeyLee** [comment]
Thanks @richardgood I have reviewed S13 RA and fixed maven parent, artifacts, versions, etc.

**Comment by brainslog** [comment]
Already merged. Thanks @richardgood!

**Pull request closed without merging**